

For my project I am using pre-existing oyster data to learn how to download data, run fastqc, align and call SNPs so that I will be able to run these analyses on my own in the future

# Repository Structure:

# Data: the .fq data files for two paired-end samples: sample1A, sample1B, sample2A, sample2B
# Analyses: fastqc results in html format for each sample (4), oyster reference genome in ebwt format as index for bowtie
# Scripts: jupyter notebook scripts for downloading the data using curl, running fastQC, and the start for a script to run botwie

# Projected Timeline:

I am currently in the process of using Bowtie to align the paired end samples to an index containing a reference oyster genome

